File: EDAII x Python/Grafos/Vertice.py
class Vertice:
	nombre = " "
	visitado = False

	def __init__(self, nombre):
		'Constructor de la clase'
		self.nombre = nombre
		self.adyacentes = []	# Crea una nueva lista vacía para cada vértice !!!
		self.aristas = []	# Crea una nueva lista vacía para cada vértice !!!

	def agregarAdyacente(self,v):
		'Agrega un nodo vecino a este'
		if len(self.adyacentes) == 0:
			self.adyacentes.append(v)
		else:
			repetido = False
			for i in range(0,len(self.adyacentes)):
				if v == self.adyacentes[i]:
					repetido = True

			if repetido != True:
				self.adyacentes.append(v)

	def imprimirAdyacentes(self):
		'Muestra en pantalla el nombre de los nodos vecinos de este'
		nombres = []
		if len(self.adyacentes) != 0:
			for i in range(0,len(self.adyacentes)):
				nombres.append(self.adyacentes[i].nombre)
			return nombres
		else:
			return "Sin nodos adyacentes"

File: EDAII x Java/Grafos/Vertice.java
import java.util.List;
import java.util.LinkedList;

public class Vertice{
	public String nombre;
	public boolean visitado=false;
	public List<Vertice> adyacentes = new LinkedList<>(); // Lista de Adyacencia
	public List<Arista> aristas = new LinkedList<>();	// Aristas que contiene el vértice
	
	public Vertice(String nombre){
		this.nombre = nombre;
	}

	public void agregarAdyacente(Vertice v){
		boolean repetido = false;
		for(int i=0; i<this.adyacentes.size(); i++){
			if(v == this.adyacentes.get(i)){
				repetido = true;
			}
		}

		if(repetido == true){
			// Nodo repetido
		} else {
			this.adyacentes.add(v);	// Si no existe el nodo, se agrega
		}
	}

	public void imprimirAdyacentes(){
		for (int i=0; i<this.adyacentes.size(); i++){
			System.out.print(this.adyacentes.get(i).nombre + " ");
		}
		System.out.println("\n");
	}

}